### DISCUSSION TITLE
录屏录播

### DISCUSSION THREAD

**Discussion by Phirida** [LocalizeLimbusCompany/LocalizeLimbusCompany#188] [想法 | Ideas]
您好，
我很喜欢你们的汉化。如果方便的话我希望你们可以把汉化的剧情做成视频分p以方便随时查看。
b站内剧情的汉化质量参差不齐，我想更多人看到零协的优质汉化。
打扰了。

**Comment by N071** [comment]
收到。我们有点疏于更新剧情视频的原因其实是觉得B站方面竞争翻译视频的up太多，与此同时大多数人其实并不在意翻译的质量，仅仅是以流量和速度作为自己选择的导向。录制剧情视频虽然并不难，但本身也是耗时的事情。再考虑到游戏本身也自带剧情播放的功能，单独再录制视频的意义可能就不那么大了——毕竟我们不追求涨粉和吸流量，并没有通过赶制视频来吸睛的压力；而我们对“翻译”和“参与社区工作”本身的渴望，其实也已经在制作模组的过程中得到了满足。
不过，既然您提到了这件事情，那么当模组文字的修改基本固定了之后，我们一定会考虑为翻译视频补档的。
